assets/app/view/create_game.rb

```diff
@@ -70,8 +70,10 @@ def render_inputs
 
       enforce_range = lambda do
         elm = Native(@inputs[:max_players]).elm
-        elm.value = elm.max.to_i unless (elm.min.to_i..elm.max.to_i).include?(elm.value.to_i)
-        store(:num_players, elm.value.to_i) if @mode == :hotseat
+        unless elm.value.empty?
```

Comment:
if elm.value.any?

Reply:
thx, I can’t remember this … (tried present? …)